### DISCUSSION TITLE
Fishing Mechanics and more (Proposal)

### DISCUSSION THREAD

**Discussion by JcEii** [sunflower-land/sunflower-land#2112] [Ideas]

**Comment by LeShykun** [comment]
Outstanding work man! Massive respect 🫡

**Comment by y-eviles** [comment]
Great work bro , i like what you proposal
big support for it.

**Comment by BirbSunflower** [comment]
Amazing proposal and art ! Good job! 😮

**Comment by vitt0** [comment]
I did some changes to the art of the island, Imo it is little bit weird that there is beach all around of a kinda small island that then has a "big cliff with a stair that takes you to a lake", I think it would be better is the island was bigger that we couldn't even see the end and full of trees. Also I changed a little bit the dock that was kinda too big Imo. I know a lot of things have some missing pixels here and there, mostly the trees, but is just to demostrate you my suggestions 😄 

About the proposal I love the idea about adding fishing, the only thing that looks kinda overwhelming Imo is that there are too many types of fishies, I know that some of them are seasonal, but anyway I think that as new player getting introduce to this is would be to much info. 

Look for example how many crops we have right now in the game (11 or 14 couting the fruit trees) you would have almost 3 times this, I think you could have half of the options that you are currently planing and still it would be plenty of different fishies to have 👍 

![Fishing](https://user-images.githubusercontent.com/11968934/215370335-90b93e8c-9046-4d62-b764-9985d9a777d5.png)

**Comment by johannesunana** [comment]
I'm still hoping this proposal pushes through. 🎣

**Comment by Floocker** [comment]
You suck you build many fish but look seed is justlittlebit400restock

**Comment by LeShykun** [comment]
rip :'(

**Reply by JcEii** [reply]
hihihihi
![image](https://github.com/sunflower-land/sunflower-land/assets/105348036/dcf9e59a-d14f-45d1-891a-26a4c1d86235)
